src/prometheus_alert_rules/unit_unavailable.rule

```diff
@@ -1,6 +1,6 @@
 alert: TraefikIngressUnitIsUnavailable
 expr: up < 1
-for: 0m
+for: 5m
```

Comment:
@dstathis is this to address the case of first install, or do you see these fire all the time?

Reply:
All the time. A few times a day. I investigated what might be wrong but I could not find any issues.